src/components/Grants/Current/CurrentGrantsBanner.tsx

```diff
@@ -1,59 +1,30 @@
 import React, { useMemo } from 'react'
 
 import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
-import isEmpty from 'lodash/isEmpty'
 
-import { GrantWithUpdateAttributes } from '../../../entities/Proposal/types'
-import { numberFormat } from '../../../modules/intl'
-import Banner, { BannerType } from '../Banner'
+import Banner, { BannerItem, BannerType } from '../Banner'
 
-const getBannerStats = (grants: GrantWithUpdateAttributes[]) => {
-  if (isEmpty(grants)) {
-    return {}
-  }
-
-  const releasedValues = grants.map((item) => {
-    const releasedPercentage = ((item.contract?.released || 0) * 100) / (item.contract?.vesting_total_amount || 0)
-
-    return ((item.size || 0) * releasedPercentage) / 100
-  })
-
-  const totalReleased = releasedValues.filter(Number).reduce((prev, next) => prev! + next!, 0) || 0
-  const toBeVestedValues = grants.map(
-    (item) => (item.contract?.vesting_total_amount || 0) - (item.contract?.vestedAmount || 0)
-  )
-  const totalToBeVested = toBeVestedValues.filter(Number).reduce((prev, next) => prev! + next!, 0) || 0
-
-  return {
-    totalGrants: grants.length,
-    totalReleased,
-    totalToBeVested,
-  }
-}
-
-const CurrentGrantsBanner = ({ grants }: { grants: GrantWithUpdateAttributes[] }) => {
+const CurrentGrantsBanner = () => {
   const t = useFormatMessage()
-  const bannerStats = useMemo(() => getBannerStats(grants), [grants])
-  const bannerItems = useMemo(
+  const bannerItems = useMemo<BannerItem[]>(
```

Comment:
If we don't have stats now we can remove this memo and this array and put all those texts directly on the render.